src/audio-worklet/free-queue/src/interface/C/free_queue.h

```diff
@@ -0,0 +1,206 @@
+#ifndef FREE_QUEUE_C
```

Comment:
This file is under `interface/C`. Having `/` in the path name doesn't sound great. Is there any reason for this?